HazelAPI/src/main/java/net/juligames/core/api/hazel/HazelObjectInterpreter.java

```diff
@@ -0,0 +1,46 @@
+package net.juligames.core.api.hazel;
+
+import com.hazelcast.core.DistributedObject;
+import com.hazelcast.core.HazelcastInstance;
+import net.juligames.core.api.config.Interpreter;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collection;
+import java.util.function.Supplier;
+
+/**
+ * @author Ture Bentzin
+ * 06.03.2023
+ */
+@ApiStatus.AvailableSince("1.5")
+public class HazelObjectInterpreter implements Interpreter<DistributedObject> {
```

Comment:
This needs further testing:
I don't know if it is possible to access the a data of the DistributedObject in this case. For this a bit of Java casting and instance behavior investigation is needed.

The review of the whole PR can continue but this file should be excluded from the final review.

Reply:
I will investigate this further after the release of 1.5! I don’t think this will cause issues and it is also very important to get this version and its associated features